src/components/About.js

```diff
@@ -57,7 +57,9 @@ class AboutDialog extends React.Component {
                 <ul className='list-unstyled'>
                   <li id={`${idPrefix}-version`}>Version <strong id={`${idPrefix}-version-value`}>{Product.version}-{Product.release}</strong></li>
                   <li id={`${idPrefix}-apiversion`}>{fixedStrings.BRAND_NAME} API Version <strong id={`${idPrefix}-apiversion-value`}>{apiVersion}</strong></li>
-                  <li id={`${idPrefix}-issues`}>Please report issues on <strong><a href='https://github.com/oVirt/ovirt-web-ui/issues' target='_blank' id={`${idPrefix}-issues-link`}>GitHub Issue Tracker</a></strong></li>
+                  {fixedStrings.ISSUES_TRACKER_URL && fixedStrings.ISSUES_TRACKER_TEXT &&
+                    <li id={`${idPrefix}-issues`}>Please report issues on <strong><a href={fixedStrings.ISSUES_TRACKER_URL} target='_blank' id={`${idPrefix}-issues-link`}>{fixedStrings.ISSUES_TRACKER_TEXT}</a></strong></li>
```

Comment:
The "Please report issues on {link}" text should probably be externalized to `src/intl/messages.js`

Reply:
i agree on the  "Please report issues on " but i dont think that you need to add the link. The link has <a>,<strong> tags and id which the "Please report issues on " dont have. and to insert it all as a json object and to pass it through a function make the code more complex instead of leave it as is, which is readable and clear